src/twisted/__main__.py

```diff
@@ -11,5 +11,5 @@
 
 if __name__ == '__main__':
     sys.exit(
-        load_entry_point('Twisted', 'console_scripts', 'twist')()
+        load_entry_point('Twisted', 'console_scripts', 'twist')()  # type: ignore[func-returns-value]  # noqa
```

Comment:
Without this change, mypy reports this error:

```
src/twisted/__main__.py:14:9: error: "load_entry_point" does not return a value  [func-returns-value]
            load_entry_point('Twisted', 'console_scripts', 'twist')()
            ^
src/twisted/trial/__main__.py:9:9: error: "load_entry_point" does not return a value  [func-returns-value]
            load_entry_point('Twisted', 'console_scripts', 'trial')()
```

This comes from setuptools: https://github.com/pypa/setuptools/blob/11051abd79b04782d759720a4d9dac2d2bbbf49c/pkg_resources/__init__.py#L488

I thought it was safer to add the comment than to change the code.

Reply:
It looks like `load_entry_point` returns some object or `None` and this is relying on the funky handling by `sys.exit` for non-`int` values…

The comment is OK… if `load_entry_point` eventually gets a type hint, we'll get an error from `mypy` that the comment is unnecessary.

Why do we also need the `noqa` comment?

Reply:
OK `noqa` is line length… I wish `mypy` had a way to comment on the next line or something.

Reply:
Yes, `noqa` is for line length.  I couldn't figure out a better way to do this when adding the mypy comment would cause the line length to exceed the length tolerated by pycodestyle.

Reply:
An alternative could be to teach pycodestyle to (optionally) strip comments for tooling when computing line length. Perhaps look for a pattern of the comment starting with a single word followed by a colon, or just ignore all comments that follow code.